pkg/manager/impl/execution_manager_test.go

```diff
@@ -287,6 +287,7 @@ func TestCreateExecution(t *testing.T) {
 			assert.Equal(t, rawOutput, spec.RawOutputDataConfig.OutputLocationPrefix)
 			assert.True(t, proto.Equal(spec.ClusterAssignment, &clusterAssignment))
 			assert.Equal(t, "launch_plan", input.LaunchEntity)
+			assert.Equal(t, spec.GetMetadata().GetSystemMetadata().Namespace, "project-domain")
```

Comment:
can we add one test with namespace mapping which equals just  `project`

Reply:
The code is already testing that we're exercising namespace mapping config but added another test